hangman.py

```diff
@@ -0,0 +1,196 @@
+# Problem Set 2, hangman.py
+# Name:Mizin Danilo
+# Collaborators:Piustonen 
+# Time spent:
+
+import string
+import random
+
+WORDLIST_FILENAME = "words.txt"
+guesses_point=6
+warnings_point=3
+letters_guessed=[]
```

Comment:
Could you avoid using global variables where it is possible to use input parameters?